src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const carPrice = 40;
+  const bigDiscount = 50;
+  const smallDiscount = 20;
+
+  if (days < 3) {
+    return days * carPrice;
+  }
+
+  if (days > 6) {
+    return days * carPrice - bigDiscount;
+  }
+
+  if (days > 2) {
+    return days * carPrice - smallDiscount;
+  }
```

Comment:
you can delete the last condition. It will default cancellation: if the number is bigger than 2 (first condition false) and smaller than 7 (second condition false))

```suggestion
  if (days < 3) {
    return days * carPrice;
  }

  if (days > 6) {
    return days * carPrice - bigDiscount;
  }

  return days * carPrice - smallDiscount;
```